translations/en.yaml

```diff
@@ -1,3 +1,5 @@
+auth.sign_up.label.workspace: Workspace Name
```

Comment:
why this long 😃

Reply:
auth.sign_up.label_workspace ? :)

Reply:
maybe two-layer is enough?  in my simple opinion...

Reply:
yes sign_up is not necessary